heron/metricscachemgr/src/java/com/twitter/heron/metricscachemgr/metricscache/store/ExceptionDatapoint.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.metricscachemgr.metricscache.store;
+
+/**
+ * immutable exception data in store
+ * TODO(huijun) object pool to avoid java gc
```

Comment:
If this is something we should really do, please open an issue for it and link to it in the comments. Same for the same comment below.